src/sentry/issues/occurrence_consumer.py

```diff
@@ -43,6 +52,92 @@ def create_ingest_occurences_consumer(
     )
 
 
+def save_event_from_occurrence(
+    data: Dict[str, Any],
+    **kwargs: Any,
+) -> Event:
+
+    from sentry.event_manager import EventManager
+
+    event_type = "platform"
```

Comment:
You'll need to inject this into the event data so that the event type changes

Reply:
whoops, good catch